rook/nodes.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+- id: setup nodes
+  uses: docker://popperized/geni:v0.9.9.2
+  args: ['./geni/setup-nodes.py']
+  secrets: [
```

Comment:
popper 2.6.0 allows to define secrets for all steps in a workflow. So we can do:

```yaml
options:
  secrets:
  - GENI_FRAMEWORK
  - ...

steps:
- id: setup nodes
  uses: docker://popperized/geni:v0.9.9.2
  args: ['./geni/setup-nodes.py']

- id: release nodes
  uses: docker://popperized/geni:v0.9.9.2
  args: ['./geni/release.py']